tables/pl.tbl

```diff
@@ -1,21 +1,10 @@
+# Based on http://liblouis.org/braille-specs/polish
 #-index-name: Polish
 #-display-name: Polish braille
 
 #+locale:pl
 #+type:literary
-#+grade:1
-# Marked as "direction:both" by Bue Vester-Andersen
-# as tests run both forward and backward
 #+direction:both
 
-# TODO: Please correct the metadata above. It is not meant to be
-# accurate nor complete. It hasn't been verified by the table
-# author yet. It is merely an attempt by the liblouis maintainers
-# to get some sensible initial values in place.
```

Comment:
Some additional metadata would be useful I think:

- contraction: (I presume this should be `#+contraction:no`).
- dots: (I presumed this should be `#+dots:6`)

Complete list of possible metadata tags is documented here: https://github.com/liblouis/liblouis/wiki/Table-discovery-based-on-table-metadata.